01_load_exported_data_and_visualize_pupillometry.ipynb

```diff
@@ -11,73 +11,49 @@
     "1. Visualize pupil diameter and pupil location as a function of time.\n",
```

Comment:
<p>@willpatera I think the output here got removed.</p><p>I would probably clean rerun all notebooks in the end once more, just to make sure that everything works fine and outputs match the code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pupil-labs/pupil-tutorials/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>@pfaion please could you re-run the notebooks and make another commit today?</p>

Reply:
Done.